extensions-core/protobuf-extensions/src/main/java/org/apache/druid/data/input/protobuf/ProtobufInputRowParser.java

```diff
@@ -89,11 +89,13 @@ public ProtobufInputRowParser withParseSpec(ParseSpec parseSpec)
       parser = parseSpec.makeParser();
     }
     Map<String, Object> record;
+    DateTime timestamp;
 
     if (parseSpec instanceof JSONParseSpec && ((JSONParseSpec) parseSpec).getFlattenSpec().getFields().isEmpty()) {
       try {
         DynamicMessage message = protobufBytesDecoder.parse(input);
         record = CollectionUtils.mapKeys(message.getAllFields(), k -> k.getJsonName());
+        timestamp = extractTimestampAsJson(record);
```

Comment:
could you fix this for the protobuf input format as well? https://github.com/apache/druid/blob/master/extensions-core/protobuf-extensions/src/main/java/org/apache/druid/data/input/protobuf/ProtobufReader.java#L87 is the relevant section for it I think

Reply:
Sure. do you have any suggestion on how to handle checkstyling for protobuf generated file?

Reply:
Ah, i guess either disabling checking, or I assume its actually ok to do what i suspect was done the last time based on the look in the diff, that someone disregarded the instructions to not modify the generated code and it was just manually auto-formatted in intellij after it was generated heh

Reply:
intellij inspections is flagging a bunch of stuff on the generated file too, hmm. trying to remember what happens if we just ignore it for now, if it will keep complaining at us in future PRs or if it only flags new things...

Reply:
I was able to fix that by annotating the class with `SuppressWarnings("ALL")`.